src/torrent.cpp

```diff
@@ -2769,9 +2769,9 @@ namespace {
 		int num_outstanding = settings().get_int(settings_pack::checking_mem_usage) * block_size()
 			/ m_torrent_file->piece_length();
 		// if we only keep a single read operation in-flight at a time, we suffer
-		// significant performance degradation. Always keep at least two jobs
+		// significant performance degradation. Always keep at least 4 jobs
 		// outstanding per hasher thread
-		int const min_outstanding = 2
+		int const min_outstanding = 4
```

Comment:
just [this](https://github.com/qbittorrent/qBittorrent/issues/9061#issuecomment-401499491)

Reply:
It would be better having own in pocket benchmark, simple with proved accuracy. What do you think on this?

Reply:
Yes, it should be more like user-oriented tool to determine optimal settings for the target environment.

Reply:
there's essentially no cost at having more outstanding hash jobs, since memory is only used while a job is executing, there's no increased memory pressure to speak of.

Reply:
Memory consumption is not a problem, what I also noticed, is that higher number of threads results in a lower CPU utilization by the client process. This could be caused either by the threads synchronization or awaiting for disk read operations.

Extra details require profiling of the client application.